RadixWallet/Features/Signing/Children/SignWithFactorSource+View.swift

```diff
@@ -0,0 +1,28 @@
+// MARK: - SignWithFactorSource.View
+public extension SignWithFactorSource {
+	@MainActor
+	struct View: SwiftUI.View {
+		private let store: StoreOf<SignWithFactorSource>
+
+		public init(store: StoreOf<SignWithFactorSource>) {
+			self.store = store
+		}
+
+		public var body: some SwiftUI.View {
+			// We need to create two different views so that each of them triggers the corresponding `.onFirstTask()` that
+			// starts the factor source access. Otherwise, when having multiple signatures chained, it would only trigger the first one.
```

Comment:
Not a big fan of this approach, but will work on an improvement later. Open to suggestions.